src/core/Microsoft.PowerToys.Settings.UI/Views/PowerPreviewPage.xaml

```diff
@@ -38,40 +38,35 @@
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto" />
         </Grid.RowDefinitions>
-        
-        <StackPanel Orientation="Vertical">
+
+        <StackPanel Orientation="Vertical" 
+                    x:Name="PowerPreviewSettingsView">
             <TextBlock Text="These settings allow you to manage your Windows File Explorer Addons."
                        TextWrapping="Wrap"/>
 
             <ToggleSwitch Header="Svg Preview Handler"
-                              x:Name="ToggleSwitch_Preview_SVG"
-                              Margin="{StaticResource SmallTopMargin}"
-                              Toggled="ToggleSwitch_Preview_SVG_Toggled" />
+                          x:Name="ToggleSwitch_Preview_SVG"
+                          Margin="{StaticResource SmallTopMargin}"
+                          IsOn="{Binding Mode=TwoWay, Path=SVGRenderIsEnebled}" />
 
             <ToggleSwitch Header="Markdown Preview Handler"
-                              x:Name="ToggleSwitch_Preview_MD"
-                              Margin="{StaticResource SmallTopMargin}"
-                              Toggled="ToggleSwitch_Preview_MD_Toggled" />
+                          x:Name="ToggleSwitch_Preview_MD"
+                          Margin="{StaticResource SmallTopMargin}"
+                          IsOn="{Binding Mode=TwoWay, Path=MDRenderIsEnebled}" />
 
         </StackPanel>
-        <StackPanel 
-            x:Name="SidePanel"
-            Orientation="Vertical"
-            HorizontalAlignment="Left"
-            Width="{StaticResource SidePanelWidth}"
-            Grid.Column="1">
+        <StackPanel x:Name="SidePanel"
+                    Orientation="Vertical"
+                    HorizontalAlignment="Left"
+                    Width="{StaticResource SidePanelWidth}"
+                    Grid.Column="1">
 
-            <TextBlock 
-                Text="About this feature"
-                Style="{StaticResource SettingsGroupTitleStyle}"
-                Margin="{StaticResource XSmallBottomMargin}"/>
+            <TextBlock Text="About this feature"
+                       Style="{StaticResource SettingsGroupTitleStyle}"
+                       Margin="{StaticResource XSmallBottomMargin}"/>
 
-            <HyperlinkButton 
-                Content="Module overview" 
-                NavigateUri="https://github.com/microsoft/PowerToys/tree/master/src/modules/previewpane"/>
-            <HyperlinkButton 
-                Content="Give feedback" 
-                NavigateUri="https://github.com/microsoft/PowerToys/issues"/>
+            <HyperlinkButton Content="Module overview" NavigateUri="https://github.com/microsoft/PowerToys/tree/master/src/modules/previewpane"/>
```

Comment:
Lets use consistent spacing as before.

Reply:
the spacing is okay. the display just can't fit to the screen.